modules/siddhi-launcher/pom.xml

```diff
@@ -0,0 +1,379 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2.siddhi.sdk</groupId>
+        <artifactId>siddhi-sdk-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>siddhi-launcher</artifactId>
+    <name>Siddhi-Launcher</name>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <description>Siddhi SDK Launcher</description>
+    <url>http://wso2.org</url>
+    <dependencies>
```

Comment:
Move the dependencies to dependecy management in parent pom and refer from there

Reply:
Resolved in commit  c50edac